### ISSUE TITLE
Invalid component (extra space) listed in component_connections.json

### ISSUE THREAD

**Issue by bgrins** [mozilla/bugbug#1983]
https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.landings_risk_report.latest/artifacts/public/component_connections.json contains both `DevTools::Inspector : Highlighters` and `DevTools::Inspector: Highlighters`. Only the latter is correct. Not sure if there are others, this one just stuck out to me.

**Comment by marco-c** [comment]
Interesting... I'll investigate where it is coming from.

**Comment by marco-c** [comment]
Here are the bugs in the DB:
```
1210632 - DevTools - Inspector : Highlighters
1665341 - DevTools - Inspector : Highlighters
1152322 - DevTools - Inspector: Highlighters
1670012 - DevTools - Inspector: Highlighters
1668265 - DevTools - Inspector: Highlighters
1635703 - DevTools - Inspector: Highlighters
1675226 - DevTools - Inspector: Highlighters
1670498 - DevTools - Inspector: Highlighters
1673561 - DevTools - Inspector: Highlighters
```

They are the only ones with ` : ` in the component string. I have no idea how that happened.
One possible explanation:
- at first, when we downloaded the first two bugs, the component was "Inspector : Highlighters";
- then, it was changed to "Inspector: Highlighters" (such changes in the Bugzilla DB don't update the last_change_time of bugs, which means bugbug won't re-download them);
- all other bugs were downloaded *after* the change.

The fact that the first are wrong and the following are good matches this explanation (bugs in the DB are store in download time order).

Anyway, I will change the bug retriever script so it automatically detects bugs with a non-existing component and re-downloads them. This should fix this and any other case that we might not notice.
